circle.yml

```diff
@@ -16,6 +16,8 @@ dependencies:
     - echo y | android update sdk --all --no-ui --force --filter "extra-android-m2repository"
     - sudo apt-get update; sudo apt-get install s3cmd
     - printf "[default]\naccess_key = $S3_ACCESS_KEY\n secret_key = $S3_SECRET_KEY" > ~/.s3cfg
+    - git clone https://github.com/mapzen/apk-s3-lister.git; cd apk-s3-lister; ./upload.sh
```

Comment:
No need to redeploy the entire download page on each build of Eraser Map. I removed this line so the site only gets deployed when the apk-s3-lister project is built by Circle.

https://circleci.com/gh/mapzen/apk-s3-lister